brainstormforce/astra
# AST-2219 - Theme package size reducing for limit of 10MB on w.org

### Description
- Removed customizer control's build JS developer version removed from the package
- 4.1 zip  - [astra-4.1.0](https://share.getcloudapp.com/6qub2Kpd)  **9.1MB** ([astra-4.1.0 (1) 2023-03-17 at 4.03.01 PM.zip Properties](https://share.getcloudapp.com/P8u2eXE9) )
- Refined [astra-4.1.0](https://share.getcloudapp.com/jkuAQRnb) - **5.86MB** ([astra-4.1.0 2023-03-17 at 4.05.08 PM.zip Properties](https://share.getcloudapp.com/z8uG8b8L) )

### Screenshots
- Attached above

### Types of changes
- Customizer control's grunt build JS version excluded from users package
- `/assets/css/unminified` excluded from final package and loaded minified assets.

### How has this been tested?
- Check customizer working correctly (both with SCRIPT_DEBUG and without mode)
- Check frontend looks appropriate in old header version + new header builder version
- Check RTL versions of frontend
- Review with supported plugins WC, EDD.

### Checklist:
